near-sdk-macros/src/lib.rs

```diff
@@ -213,6 +213,140 @@ pub fn metadata(item: TokenStream) -> TokenStream {
     }
 }
 
+#[proc_macro_derive(NearAbi, attributes(abi, serde, borsh_skip, schemars, validate))]
```

Comment:
The only thing about this name is that it doesn't fit in with naming of traits. Generally, they are named as verbs, so `NearAbi` might be a bit confusing, especially since it's exported from the SDK root that it might be assumed to be the Abi schema type.

@itegulov do you have any ideas for naming? I would be fine with this, just want to see if there could be something better/more clear.

Reply:
`NearSchema` maybe? Fits more appropriately with the traits it aliases.

Reply:
Hmm, I know I was the one who suggested `NearAbi` with `#[abi(json)]`, but now I kind of feel like two separate `NearJsonSchema` & `NearBorshSchema` trait would feel more natural. Otherwise yeah `NearSchema` seems appropriate. Sorry for re-raising this point, definitely not a big deal either way.